### PR TITLE
[terra-icon] OCS icons v1.34.0

### PR DESCRIPTION
**Pull request by jeremyfuksa**
### Summary

* Bumped `one-cerner-style-icons` to v1.34.0
* Updated deprecated Mocha test assertion helpers

### CODE REVIEW

File: packages/terra-icon/tests/wdio/icon-spec.js [hunk 1]
```diff
@@ -2,30 +2,40 @@ Terra.describeViewports('Icon', ['huge'], () => {
...
+      Terra.validates.element();
+    });
   });
 
   describe('HeightWidth', () => {
     before(() => browser.url('/#/raw/tests/terra-icon/icon/icon-height-width'));
 
-    Terra.it.validatesElement();
+    it('checks element', () => {
+      Terra.validates.element();
```
Comment (emilyrohrbough) [inline packages/terra-icon/tests/wdio/icon-spec.js:14]:
```suggestion
    it('checks element', () => {
      browser.url('/#/raw/tests/terra-icon/icon/icon-height-width');
      Terra.validates.element();
```

File: packages/terra-icon/tests/wdio/icon-spec.js [hunk 2]
```diff
@@ -2,30 +2,40 @@ Terra.describeViewports('Icon', ['huge'], () => {
...
     before(() => browser.url('/#/raw/tests/terra-icon/icon/icon-default'));
 
-    Terra.it.validatesElement();
+    it('checks element', () => {
+      Terra.validates.element();
+    });
   });
 
   describe('HeightWidth', () => {
     before(() => browser.url('/#/raw/tests/terra-icon/icon/icon-height-width'));
```
Comment (emilyrohrbough) [inline packages/terra-icon/tests/wdio/icon-spec.js:11]:
```suggestion
```

File: packages/terra-icon/tests/wdio/icon-spec.js [hunk 3]
```diff
@@ -2,30 +2,40 @@ Terra.describeViewports('Icon', ['huge'], () => {
   describe('Default', () => {
```
Comment (mjhenkes) [inline packages/terra-icon/tests/wdio/icon-spec.js:2]: @ryanthemanuel are these tests auto generated?

File: packages/terra-icon/src/IconHazardLow.module.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+@import './orion-fusion-theme/IconHazardLow.module';
+@import './clinical-lowlight-theme/IconHazardLow.module';
+
+/* stylelint-disable selector-class-pattern */
+:local {
+  .IconHazardLow {
+    circle {
+      fill: var(--terra-icon-hazard-low-circle-fill,#0092e0);
```
Comment (mjhenkes) [inline packages/terra-icon/src/IconHazardLow.module.scss:8]: Build is failing here:
```
packages/terra-icon/src/IconHazardLow.module.scss
 8:52  ✖  Expected single space after "," in a single-line function   function-comma-space-after
```
Easy fix, but was this generated? do we need to fix it somewhere else too?
```suggestion
      fill: var(--terra-icon-hazard-low-circle-fill, #0092e0);
```

File: packages/terra-icon/tests/wdio/icon-spec.js (hunk 3 above)
Comment (jeremyfuksa) [reply packages/terra-icon/tests/wdio/icon-spec.js:2]: I made these updates.

**Review by mjhenkes — APPROVED** [review]

File: packages/terra-icon/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Bumped `one-cerner-style-icons` to v1.34.0
```
Comment (StephenEsser) [inline packages/terra-icon/CHANGELOG.md:5]:
```suggestion
* Changed
  * Bumped `one-cerner-style-icons` to v1.34.0
```

**Review by emilyrohrbough — APPROVED** [review]

**Pull request merged**